### PR TITLE
Add a notification button on the header for contacts and organizations

### PR DESCRIPTION
**Pull request by JamesSCBD**

### CODE REVIEW

File: components/kronos/header/DropdownContacts.vue [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        <CHeaderNavLink>
+          <CIcon name="addressBook" />
+          <CBadge v-if="showSelectionCount" color="info" class="ml-auto">
+            {{ showSelectionCount }}
+          </CBadge>
+        </CHeaderNavLink>
+      </CHeaderNavItem>
+    </template>
+    <CDropdownItem :disabled="!selectedCount" @click="clearSelection()">
+      <i class="cil-clear-all pr-1" />Clear Selection
```
Comment (sbilodeau) [inline components/kronos/header/DropdownContacts.vue:19]: add the count:
> Clear selection:  34 records

File: components/kronos/header/DropdownContacts.vue [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    </CDropdownItem>
+    <CDropdownItem :disabled="!selectedQueryCount" @click="clearQuery()">
+      <i class="cil-clear-all pr-1" />Clear Query
```
Comment (sbilodeau) [inline components/kronos/header/DropdownContacts.vue:22]: add the count:
> Clear selection query:  2658 records

File: store/contacts.js [hunk 3]
```diff
@@ -11,6 +11,8 @@ const ADD_TO_SELECTION      = 'ADD_TO_SELECTION';
 const REMOVE_FROM_SELECTION = 'REMOVE_FROM_SELECTION';
 const CLEAR                 = 'CLEAR';
 const SAVE_SELECTED_QUERY   = 'SAVE_SELECTED_QUERY';
+const CLEAR_SELECTION       = 'CLEAR_SELECTION';
+const CLEAR_QUERY           = 'CLEAR_QUERY';
```
Comment (sbilodeau) [inline store/contacts.js:15]: rename to `CLEAR_SELECTION_QUERY`

File: store/contacts.js [hunk 4]
```diff
@@ -91,6 +104,14 @@ const $actions = {
     const count    = response.totalRecordCount;
     commit(SAVE_SELECTED_QUERY, { query, count });
   },
+
+  clearSelection({ commit }) {
+    commit(CLEAR_SELECTION);
+  },
+
+  clearQuery({ commit }) {
```
Comment (sbilodeau) [inline store/contacts.js:112]: rename to clearSelectionQuery

File: store/organizations.js [hunk 5]
```diff
@@ -15,6 +15,8 @@ const ADD_TO_SELECTION      = 'ADD_TO_SELECTION';
 const REMOVE_FROM_SELECTION = 'REMOVE_FROM_SELECTION';
 const CLEAR                 = 'CLEAR';
 const SAVE_SELECTED_QUERY   = 'SAVE_SELECTED_QUERY';
+const CLEAR_SELECTION       = 'CLEAR_SELECTION';
+const CLEAR_QUERY           = 'CLEAR_QUERY';
```
Comment (sbilodeau) [inline store/organizations.js:19]: raname to `CLEAR_SELECTION_QUERY`

File: store/organizations.js [hunk 6]
```diff
@@ -133,6 +145,14 @@ const $actions = {
     const count    = response.totalRecordCount;
     commit(SAVE_SELECTED_QUERY, { query, count });
   },
+
+  clearSelection({ commit }) {
+    commit(CLEAR_SELECTION);
+  },
+
+  clearQuery({ commit }) {
```
Comment (sbilodeau) [inline store/organizations.js:153]: rename to `clearSelectionQuery`

File: components/kronos/header/DropdownOrganizations.vue [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  },
+  computed: {
+    showSelectionCount() {
+      if (this.selectedCount && this.selectedQueryCount) {
+        return `${this.selectedCount} / ${this.selectedQueryCount}`;
+      } if (this.selectedCount) {
+        return `${this.selectedCount}`;
+      } if (this.selectedQueryCount) {
+        return `${this.selectedQueryCount}`;
+      }
```
Comment (sbilodeau) [inline components/kronos/header/DropdownOrganizations.vue:46]: There is no `else` please move if on a new line not right after `}`
```suggestion
      } 
      if (this.selectedCount) {
        return `${this.selectedCount}`;
      } 
      if (this.selectedQueryCount) {
        return `${this.selectedQueryCount}`;
      }
```

File: components/kronos/header/DropdownContacts.vue [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+  },
+  computed: {
+    showSelectionCount() {
+      if (this.selectedCount && this.selectedQueryCount) {
+        return `${this.selectedCount} / ${this.selectedQueryCount}`;
+      } if (this.selectedCount) {
+        return `${this.selectedCount}`;
+      } if (this.selectedQueryCount) {
+        return `${this.selectedQueryCount}`;
+      }
```
Comment (sbilodeau) [inline components/kronos/header/DropdownContacts.vue:46]: There is no else please move if on a new line not right after }

```suggestion
      }
      if (this.selectedCount) {
        return `${this.selectedCount}`;
      }
      if (this.selectedQueryCount) {
        return `${this.selectedQueryCount}`;
      }
```

Comment (sbilodeau) [reply components/kronos/header/DropdownContacts.vue:46]: if the linter complains. add an empty line between

File: components/kronos/header/DropdownContacts.vue (hunk 1 above)
Comment (JamesSCBD) [reply components/kronos/header/DropdownContacts.vue:19]: Count display in `Badge` for nice look

File: components/kronos/header/DropdownContacts.vue (hunk 2 above)
Comment (JamesSCBD) [reply components/kronos/header/DropdownContacts.vue:25]: Count display in `Badge` for nice look

File: store/contacts.js (hunk 3 above)
Comment (JamesSCBD) [reply store/contacts.js]: Done!

File: store/contacts.js (hunk 4 above)
Comment (JamesSCBD) [reply store/contacts.js]: Done!

File: store/organizations.js (hunk 5 above)
Comment (JamesSCBD) [reply store/organizations.js]: Done!

File: store/organizations.js (hunk 6 above)
Comment (JamesSCBD) [reply store/organizations.js]: Done!

File: components/kronos/header/DropdownOrganizations.vue (hunk 7 above)
Comment (JamesSCBD) [reply components/kronos/header/DropdownOrganizations.vue]: Done!

File: components/kronos/header/DropdownContacts.vue (hunk 8 above)
Comment (JamesSCBD) [reply components/kronos/header/DropdownContacts.vue]: Done!

**Pull request merged**